### DISCUSSION TITLE
Unable to Exclude Routes/Endpoints from Generated Documentation

### DISCUSSION THREAD

**Discussion by sefakor20** [dedoc/scramble#298] [Q&A] [answered]
I am currently using dedoc/scramble to generate API documentation for my Laravel project. However, I am facing an issue where I am unable to exclude certain routes or endpoints from the generated documentation. 

I am looking for some guidance or assistance on how to properly exclude these routes or endpoints. Any help would be greatly appreciated. Thank you!

**Comment by sumanWebskitters** [comment]
@sefakor20 any luck??

**Reply by romalytvynenko** [reply, marked as answer]
@sefakor20 @sumanWebskitters 

https://scramble.dedoc.co/usage/getting-started#excluding-hiding-some-routes-from-api-documentation

**Comment by pkaruga** [comment]
@romalytvynenko how do I exclude endpoints in an extended controller?
